src/listener/Heading.php

```diff
@@ -27,10 +27,19 @@ public function process(Line $line)
 
     /**
      * {@inheritDoc}
+     * 
+     * @since 1.2.0 Added exception
+     * 
+     * @throws \Exception for unknown heading levels
      */
     public function render(\nadar\quill\Lexer $lexer)
     {
         foreach ($this->picks() as $pick) {
+            if (in_array($pick->heading, [1, 2, 3, 4, 5, 6], $strict=true) === false) {
+                // prevent html injection in case the attribute is user input
+                throw new \Exception('unknown heading level');
```

Comment:
Please add Exception to use statements instead of inline definition. Maybe change exception to start with Uppercase.

```
An unknown heading level {$level} has been detected..